src/example/App/VariableText.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import createReactClass from 'create-react-class';
 import {shouldComponentUpdate} from 'react/lib/ReactComponentWithPureRenderMixin';
```

Comment:
Yeah, that is cool. Won't mind a PR for this. Currently it does not produce any warnings.

There was a very practical reason to deeplink it.

React team did not create a standalone package for shallow compare so we could not make it `external`. If we used special package - it just required this deeplinked `react/lib/ReactComponentWithPureRenderMixin`. So when included in the bundle - well... You include half of React package with it. Not nice for a small library.

Bundle size should not be increased when we migrated to whatever other solution is there...